experiments/golden-results/SPARK-tetris-summary.txt

```diff
@@ -13,9 +13,9 @@ Calling function: Process_Pragma_Declaration
 Error message: Unsupported pragma: Volatile
 Nkind: N_Pragma
 --
-Occurs: 2 times
+Occurs: 1 times
 Calling function: Process_Declaration
-Error message: Representation clause declaration
+Error message: Representation clause unsupported: component_size
```

Comment:
How is this fix related to the change in this PR?  It seems vaguely like the change in the golden results for the `'Size` patch...